src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java

```diff
@@ -149,29 +142,29 @@ private Vocabulary resolveVocabularyFromGlossary(final String vocabularyIri) {
         labels.stream().filter(s -> {
             assert s.getObject() instanceof Literal;
             return Objects.equals(config.get(ConfigParam.LANGUAGE),
-                ((Literal) s.getObject()).getLanguage().orElse(config.get(ConfigParam.LANGUAGE)));
+                    ((Literal) s.getObject()).getLanguage().orElse(config.get(ConfigParam.LANGUAGE)));
```

Comment:
Are you using default Idea formatting?

Reply:
Yes I am. I am just looking into defining an .editorconfig file to prevent future issues with formatting.

Reply:
Sounds good.